Example/Alps/MatchDelegate.swift

```diff
@@ -8,10 +8,11 @@
 
 import Foundation
 import AlpsSDK
+import Alps
 
 class MatchDelegate: AlpsManagerDelegate {
-    var onMatch: OnMatchClojure
-    init(_ onMatch: @escaping OnMatchClojure) {
```

Comment:
You deleted my little joke lol

Reply:
I remember when reading this line I was like: "When did that happen?" :)